src/test/java/com/mdudzisz/ticketbookingapp/service/TimeIntervalTest.java

```diff
@@ -0,0 +1,95 @@
+package com.mdudzisz.ticketbookingapp.service;
+
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.sql.Timestamp;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class TimeIntervalTest {
+
+    @Test
+    public void fromQueryMap_toGreaterFrom() {
```

Comment:
Ten test mi nie przeszedł, bo w momencie odpalenia jest już czerwiec :) 
1. Jak byś zadbał o to, żeby testy przechodziły niezależnie od czasu uruchomienia?
2. Jaki mechanizm można wykorzystać, żeby nie dało się zmergować kodu, w którym testy nie przechodzą?